framework/decode/dx12_dump_resources.h

```diff
@@ -60,6 +60,13 @@ enum class Dx12DumpResourceType : uint32_t
     kComputeRootParameters,
 };
 
+const uint64_t modifiableTransitionStates =
+    D3D12_RESOURCE_STATE_RENDER_TARGET | D3D12_RESOURCE_STATE_DEPTH_WRITE | D3D12_RESOURCE_STATE_STREAM_OUT |
+    D3D12_RESOURCE_STATE_COPY_DEST | D3D12_RESOURCE_STATE_RESOLVE_DEST |
+    D3D12_RESOURCE_STATE_RAYTRACING_ACCELERATION_STRUCTURE | D3D12_RESOURCE_STATE_ALL_SHADER_RESOURCE |
+    D3D12_RESOURCE_STATE_VIDEO_DECODE_WRITE | D3D12_RESOURCE_STATE_VIDEO_PROCESS_WRITE |
+    D3D12_RESOURCE_STATE_VIDEO_ENCODE_WRITE;
+
```

Comment:
Is there any document to show what states are modifiable?

Reply:
`D3D12_RESOURCE_STATE_COMMON` is a special state which also may be writeable. A resource state can be promoted from `D3D12_RESOURCE_STATE_COMMON` to other resource states--both read or write states--depending on how the resource is used.